src/blog/2024/12/why-uns-need-data-modeling.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "UNS & Why You Need Data Modeling"
+subtitle: Why data modeling is key to making your Unified Namespace work effectively.
+description: Discover why data modeling is crucial for a Unified Namespace (UNS) in manufacturing and how it helps organize and make data actionable.
+date: 2024-12-04
+authors: ["sumit-shinde"]
+image: 
+keywords: uns, data modeling, why uns needs data modeling, unified namespace
+tags:
+   - flowfuse
+   - uns
```

Comment:
Hi @sumitshinde-84,  

We now have a blog page dedicated to UNS articles (check https://github.com/FlowFuse/website/pull/2855). To have articles added to this collection, they should include the `unified-namespace` tag.  

You can find the list of tags in the [handbook](https://flowfuse.com/handbook/customer/marketing/blog/#tags).

```suggestion
   - unified-namespace
```

Reply:
Thanks Yndira, I will update it for all our latest uns blogs